OpenRA.Mods.Common/Activities/Move/Move.cs

```diff
@@ -315,12 +317,13 @@ public override void Queue(Activity activity)
 
 			public override Activity Tick(Actor self)
 			{
+				var mobile = self.Trait<Mobile>();
```

Comment:
You should probably cache this as well in the constructor. All the callers are in Move.cs anyway, and `Move` has it cached already has well, so it can just pass it in.